tests/cmdline/commands/test_group.py

```diff
@@ -17,17 +17,19 @@
 class TestVerdiGroup(AiidaTestCase):
     """Tests for the `verdi group` command."""
 
-    @classmethod
-    def setUpClass(cls, *args, **kwargs):
-        super().setUpClass(*args, **kwargs)
-        for group in ['dummygroup1', 'dummygroup2', 'dummygroup3', 'dummygroup4']:
-            orm.Group(label=group).store()
-
     def setUp(self):
         """Create runner object to run tests."""
         from click.testing import CliRunner
         self.cli_runner = CliRunner()
 
+        for group in ['dummygroup1', 'dummygroup2', 'dummygroup3', 'dummygroup4']:
+            orm.Group(label=group).store()
+
+    def tearDown(self):
```

Comment:
Is the DB not reset in between tests?
anyhow, it doesn't hurt...

Reply:
only between test class instances